src/chrome/content/rules/123systems.xml

```diff
@@ -1,4 +1,9 @@
+
 <!--
+Disabled by https-everywhere-checker because:
+Fetch error: http://123systems.net/ => https://123systems.net/: Too many redirects while fetching 'https://123systems.net/'
+Fetch error: http://www.123systems.net/ => https://www.123systems.net/: Too many redirects while fetching 'https://www.123systems.net/'
+
```

Comment:
The bare host 123systems.net is not excluded. There is an exclusion tag but it is commented out.